core/commands/public_keys_command.rb

```diff
@@ -41,6 +36,22 @@ def show_help
 
   private
 
+  # Сopies the ssh key to available nodes.
+  def copy_key_to_node
+    available_nodes = @network_config.store_network_config
+    if available_nodes.empty?
+      @ui.error('No available nodes')
+      return ERROR_RESULT
+    else
```

Comment:
No need for the else clause because we will return if the list of available nodes is empty.